packages/slice-machine/lib/models/common/widgets/Date/Mock/Form.js

```diff
@@ -1,20 +1,19 @@
 import TimeConfigForm from "components/TimeConfigForm";
-
-import { initialValues } from ".";
+import { DefaultConfig } from "@lib/mock/LegacyMockConfig";
 
 const Form = (props) => {
   return (
     <TimeConfigForm
       {...props}
       // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-      initialMockValues={initialValues}
+      initialMockValues={DefaultConfig.Date}
```

Comment:
are we sure that we need this one ?
I mean we set it a couple of lines below so it feels weird to set it twice.
+ it doesn't seem that we set it twice on the other widgets.

Reply:
this is how it's build and I didn't intend to change how the form behave so I kept the exact same setup.

Reply:
But don't you think it is weird ?

Reply:
It might but I can't fix all SM code in my PR, there are a bunch of clunky places and I didn't intend to fix them all